perun-openapi/openapi.yml

```diff
@@ -5135,7 +5279,7 @@ paths:
       operationId: getFacilityByName
       summary: Returns Facility with specified name.
       parameters:
-        - { name: name, description: "name of Facility", schema: { type: string }, in: query, required: true}
+        - $ref: '#/components/parameters/name'
```

Comment:
You should not add changes not relevant to the current problem you are dealing with. You should create another PR, or at least if you do so, add a well-explaining commit message.